.github/workflows/ci-build-release-napi.yml

```diff
@@ -41,13 +41,16 @@ jobs:
       - name: Build CPP dependencies lib
         if: steps.cache-dependencies.outputs.cache-hit != 'true'
         run: |
-          export ARCH=${{ matrix.arch }}
+          if [ "${{ matrix.arch }}" = "x64" ]; then
+            export ARCH=x86_64
+          else
+            export ARCH=${{ matrix.arch }}
+          fi
```

Comment:
It confuses me that we changed `arch` to `x64` above be resolved it to `x86_64` again here - what's the difference?

Reply:
Because in `build-cpp-deps-lib.sh`, we use `x86_64` parameters uniformly. Used for configuration `CFLAGS` or `CXXFLAGS`

https://github.com/apache/pulsar-client-node/blob/10ef71a0d235955d91bf763d12513eca51948df0/pkg/mac/build-cpp-deps-lib.sh#L150-L151